include/megaapi.h

```diff
@@ -3311,6 +3311,7 @@ class MegaRequest
          * - MegaApi::setUserAttribute - Returns the attribute type
          * - MegaApi::getUserAttribute - Returns the attribute type
          * - MegaApi::setMaxConnections - Returns the direction of transfers
+         * - MegaApi::dismissBanner - Returns the id of the banner
```

Comment:
It's great you updated it here (despite we are not keeping the list up to date anymore). We usually document only the method in MegaApi, with the information about the members in the request that are valid. Not the other way.
Just for your information, no need to change it